content/docs/docs-nav.yaml

```diff
@@ -6,14 +6,14 @@ plugins:
 
 nav:
   - Getting started:
-    - Installing the GitHub app: '/docs/getting-started/install-github-app/'
-    - Adding components: '/docs/getting-started/adding-components/'
-    - Using TechDocs: '/docs/getting-started/technical-documentation/'
-    - Adding plugins: '/docs/getting-started/configuring-backstage-plugins/'
-    - Inviting users: '/docs/getting-started/user-management/'
-    - Designating admins: '/docs/getting-started/create-admin-group/'
-    - Adding API specs: '/docs/getting-started/openapi-specs/'
-    - Scaffolding components: '/docs/getting-started/scaffolding-components/'
+      - Installing the GitHub app: '/docs/getting-started/install-github-app/'
+      - Adding components: '/docs/getting-started/adding-components/'
+      - Using TechDocs: '/docs/getting-started/technical-documentation/'
+      - Adding plugins: '/docs/getting-started/configuring-backstage-plugins/'
+      - Inviting users: '/docs/getting-started/user-management/'
+      - Designating admins: '/docs/getting-started/create-admin-group/'
+      - Adding API specs: '/docs/getting-started/openapi-specs/'
+      - Scaffolding components: '/docs/getting-started/scaffolding-components/'
```

Comment:
What is this diff for? Seems like these lines should be untouched. Is this a tabs vs. spaces thing?